FluentUI/FluentUIStyleSheets/GenericStyle.yml

```diff
@@ -0,0 +1,236 @@
+  ringDefaultColor: "FluentUIColor(light: $Colors.Brand.tint10, dark: $Colors.Neutral.grey78)"
+
+OverflowAvatarTokens extends AvatarTokens:
+  backgroundDefaultColor: $Colors.Background.neutral4
+  foregroundDefaultColor: $Colors.Foreground.neutral3
+  textColor: $Colors.Foreground.neutral3
+  ringDefaultColor: $Colors.Background.neutralDisabled
+
+#FIXME: Naming and enums differ from iOS
+AP_ButtonTokens:
+  ButtonSize: EnumDef(small, medium, large)
+  ButtonStyle: EnumDef(primary, secondary, ghost)
+  backgroundColor: [ rest: $Colors.Neutral.clear, hover: $Colors.Neutral.clear, pressed: $Colors.Neutral.clear, selected: $Colors.Neutral.clear, disabled: $Colors.Neutral.clear ]
+  borderColor: [ rest: $Colors.Background.brandRest, hover: $Colors.Background.brandHover, pressed: $Colors.Background.brandPressed, selected: $Colors.Background.brandSelected, disabled: $Colors.Background.brandDisabled ]
+  borderRadius: [ small: $Border.radius.medium, medium: $Border.radius.large, large: $Border.radius.xlarge ]
+  borderSize: [ small: $Border.size.none, medium: $Border.size.none, large: $Border.size.none ]
+  iconColor: [ rest: $Colors.Foreground.brandRest, hover: $Colors.Foreground.brandHover, pressed: $Colors.Foreground.brandPressed, selected: $Colors.Foreground.brandSelected, disabled: $Colors.Foreground.brandDisabled ]
+  iconSize: [ small: $Icon.size.xSmall, medium: $Icon.size.small, large: $Icon.size.small ]
+  interspace: [ small: $Spacing.xxSmall, medium: $Spacing.xSmall, large: $Spacing.xSmall ]
+  padding: [ small: $Spacing.xSmall, medium: $Spacing.small, large: $Spacing.large ]
+  textColor: [ rest: $Colors.Foreground.brandRest, hover: $Colors.Foreground.brandHover, pressed: $Colors.Foreground.brandPressed, selected: $Colors.Foreground.brandSelected, disabled: $Colors.Foreground.brandDisabled ]
+  textFont: [ small: $Typography.footnote, medium: $Typography.subheadline, large: $Typography.subheadline ]
+
+PrimaryButtonTokens extends ButtonTokens:
+  backgroundColor: [ rest: $Colors.Background.brandRest, hover: $Colors.Background.brandHover, pressed: $Colors.Background.brandPressed, selected: $Colors.Background.brandSelected, disabled: $Colors.Background.brandDisabled ]
+  iconColor: [ rest: $Colors.Foreground.neutralInverted, hover: $Colors.Foreground.neutralInverted, pressed: $Colors.Foreground.neutralInverted, selected: $Colors.Foreground.neutralInverted, disabled: $Colors.Foreground.neutralInverted ]
+  textColor: [ rest: $Colors.Foreground.neutralInverted, hover: $Colors.Foreground.neutralInverted, pressed: $Colors.Foreground.neutralInverted, selected: $Colors.Foreground.neutralInverted, disabled: $Colors.Foreground.neutralInverted ]
+
+SecondaryButtonTokens extends ButtonTokens:
+  borderColor: [ rest: $Colors.Stroke.brandRest, hover: $Colors.Stroke.brandHover, pressed: $Colors.Stroke.brandPressed, selected: $Colors.Stroke.brandSelected, disabled: $Colors.Stroke.brandDisabled ]
```

Comment:
In Teams we are working on mapping colors to the current Fluent set? Will the new Fluent UI introduce new colors or is the "yml" content based on the current color palette.